### PR TITLE
Use openshift-installer for cluster creation and deletion

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

**Review by myakove — CHANGES_REQUESTED** [review]
please change 
BASIC_LOGGER to `BASIC_LOGGER = logging.getLogger("basic")` in conftest.py

File: tests/hypershift/test_hypershift.py [hunk 1]
```diff
@@ -314,5 +137,11 @@ def test_uninstall_operator(self, cluster_scope_class):
...
-    def test_hypershift_cluster_uninstall(self, cluster_scope_class):
-        cluster_scope_class.delete()
+    def test_hypershift_cluster_uninstall(self, destroy_cluster_cmd, click_runner):
+        result = click_runner.invoke(
+            cli=openshift_cli_installer.cli.main,
+            args=destroy_cluster_cmd,
+            catch_exceptions=False,
+        )
+        if result.exit_code != 0:
+            pytest.fail(f"Failed to create cluster on {result.output}")
```
[the remark this answers is not in the document]
Comment (myakove) [reply tests/hypershift/test_hypershift.py:147]: pytest.fail(f"Failed to destroy cluster. error:  {result.output}")

**Review by myakove — CHANGES_REQUESTED** [review]

File: tests/hypershift/test_hypershift.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (rnetser) [reply tests/hypershift/test_hypershift.py]: done

**Review by myakove — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**